mlflow/protos/mlflow_artifacts.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto2";
+
+package mlflow.artifacts;
+
+import "scalapb/scalapb.proto";
+import "databricks.proto";
```

Comment:
do we need to import `databricks.proto`?

Reply:
It appears `databricks.proto` is required to use `option (rpc)`: 

```
$ ./generate-protos.sh

++ protoc --version
+ PROTOC_VERSION='libprotoc 3.6.0'
+ [[ libprotoc 3.6.0 != \l\i\b\p\r\o\t\o\c\ \3\.\6\.\0 ]]
+ PROTOS=mlflow/protos
+ protoc -I=mlflow/protos --python_out=mlflow/protos --java_out=mlflow/java/client/src/main/java mlflow/protos/databricks.proto mlflow/protos/service.proto mlflow/protos/model_registry.proto mlflow/protos/databricks_artifacts.proto mlflow/protos/mlflow_artifacts.proto mlflow/protos/scalapb/scalapb.proto
mlflow_artifacts.proto:16:12: Option "(rpc)" unknown.
mlflow_artifacts.proto:28:12: Option "(rpc)" unknown.
mlflow_artifacts.proto:40:12: Option "(rpc)" unknown.
```

Reply:
Got it! Thanks!